### DISCUSSION TITLE
while working with self hosted api gateway along with aks

### DISCUSSION THREAD

**Discussion by kirankumarsivapuram** [Azure/api-management-self-hosted-gateway#247] [Q&A]
Can any one please help me in solving the below issue
Getting problem while implementing self hosted gate way using Api management with AKS
Steps followed
1) Created aks cluster
2)created apim in the same resource group
3) created gateway with existed api (echo api)
4)selected the gate way and selected deployment and selected the deployment script Kubernetes
downloaded the given yaml file and ran the download file on aks but i am facing the below problems
i) unable to run the pod
ii) unable to see the nodes in the apimanagement service

**Comment by tomkerkhove** [comment]
Opening a support ticket for this might be best though.

You're unable to see the instances in the portal because apparently you have challenges running the pod.

Can you elaborate on the problem you are having please?
